### ISSUE TITLE
Playing local songs also adds soundcloud songs

### ISSUE THREAD

**Issue by nolyn** [mopidy/mopidy-soundcloud#50]
Hi,
I also use the local backend of mopidy. When I add an artist from the local library, the playlist also fills up with songs found in soundcloud. I'd like to only play the local ones however. If I want to play something from soundcloud, I use search. Is it possible to only play the local songs? I have mopidy running on a raspberry pi, I tried MPDroid and GMPC as clients.

**Comment by alberdingk-thijm** [comment]
I think I'm seeing the same thing in `ncmpcpp`: browsing the media library, I'm getting back what appear to be soundcloud query results in addition to local album tracks, even when not looking in the soundcloud album. Here's what I mean:
![mopidy-soundcloud](https://user-images.githubusercontent.com/10522804/39222812-9ba5d96a-480c-11e8-869d-c8be3c685f8e.png)

**Comment by tylerball** [comment]
Agreed, it would be awesome to be able to play things from soundcloud on mopidy but exclude it's results from browse as there's a lot of mis-tagged garbage that shows up.
